### ISSUE TITLE
question about the number of output_projection

### ISSUE THREAD

**Issue by violet-sto** [microsoft/torchscale#29]
Hello!

I found that there is only one output_projection (nn.Linear(768, 64000)) for masked language modeling. However, as Beit-3 is a multimodal model, should there also be a output head for masked image modeling?

**Comment by donglixp** [comment]
Yes, we used separate heads.
